src/extension/_locales/fr/messages.json

```
@@ -117,7 +117,7 @@
     "message": "URL de partage incorrecte"
   },
   "config_invalid_giturl": {
-    "message": "URL du code dans GitHub incorrecte"
+    "message": "URL de référentiel non valide, doit commencer par https://github.com"
```

I would simplify the message to: 
```suggestion
    "message": "L'URL du projet doit commencer par https://github.com"
```